### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by radiac**
This fixes a couple of typos, sets a page title (`<head><title>...`) value for notebook-based pages, and restyles the blocks page so the title comes first.

May be worth noting that the changes in 05942fa were entirely from building the docs - let me know if I should get rid of this commit.

### CODE REVIEW

File: docs/mkdocs.yml [hunk 1]
```diff
@@ -13,9 +13,9 @@ nav:
       - Install Datapane: "tutorials/basics/install-datapane.md"
       - Create a report: "tutorials/basics/create-a-report.ipynb"
     - 🧱 Blocks types:
-      - tutorials/blocks/index.ipynb
+      - Block types: tutorials/blocks/index.ipynb
```
Comment (shahinrostami) [inline docs/mkdocs.yml:16]: Interesting change do the section indexes in `mkdocs.yml`! We had followed the [docs](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#section-index-pages) for this, is this a better way of doing it?

File: docs/docs/catalogues/blocks/index.ipynb [hunk 2]
```diff
@@ -25,17 +25,17 @@
     "\n",
     "<div class=\"grid cards\" markdown>\n",
     "    \n",
-    "-   <a href=\"/catalogues/blocks/table\"><img style=\"width:100%;\" src=\"../../img/catalogues/table.png\"></a>\n",
+    "-   __[Table](/catalogues/blocks/table)__\n",
```
Comment (shahinrostami) [inline docs/docs/catalogues/blocks/index.ipynb:28]: What was the rationale behind switching titles/images? We had them in the more common order (newspaper/print/website cards)

**Review by shahinrostami — COMMENTED** [review]
Nice, @radiac ! 

The docs are built after they're merged - so removing that commit is fine!

I've left a couple of comments 👍

File: docs/mkdocs.yml (hunk 1 above)
Comment (radiac) [reply docs/mkdocs.yml:16]: I found the `.ipynb` files weren't setting their page titles (eg [blocks](https://docs.datapane.com/catalogues/blocks/)) - a bug in  ``mknotebooks`` perhaps? The easiest way to fix it seemed to be by setting the index page titles manually.

File: docs/docs/catalogues/blocks/index.ipynb (hunk 2 above)
Comment (radiac) [reply docs/docs/catalogues/blocks/index.ipynb:28]: I wasn't sure if this was intentional. I'd first read these pages on mobile, and found the styling on [blocks](https://docs.datapane.com/catalogues/blocks/) of img/title to be a bit confusing, but clearer on [components](https://docs.datapane.com/catalogues/components/) where it's tile/img. I think img/title would be fine, but could do with a bit more styling to associate them - max-height the image and group them in a visual container perhaps. Or just swap their order on mobile with css?

**Comment by mands** [comment]
Should we stripe the notebook metadata first, and create a pre-commit hook to always strip as part of this merge?

**Comment by shahinrostami** [comment]
Hey @radiac - shall we close this PR?

**Comment by radiac** [comment]
Yes - I'll sort out a new one next week.

**Pull request closed without merging**